packages/now-cli/src/commands/inspect.js

```diff
@@ -101,14 +99,13 @@ export default async function main(ctx) {
 
   // resolve the deployment, since we might have been given an alias
   const depFetchStart = Date.now();
-  const cancelWait = output.spinner(
+  output.spinner(
     `Fetching deployment "${deploymentIdOrHost}" in ${chalk.bold(contextName)}`
   );
 
   try {
     deployment = await now.findDeployment(deploymentIdOrHost);
   } catch (err) {
```

Comment:
Do you need to stop this spinner here?

Reply:
No, because calling `error()` below implicitly clears the spinner.